contracts/hermez/future_versions/lib/HermezHelpersV2.sol

```diff
@@ -200,120 +200,190 @@ contract HermezHelpersV2 is Initializable {
             // if length < 32 bytes so let's prepare for that
             // v. http://solidity.readthedocs.io/en/latest/miscellaneous.html#layout-of-state-variables-in-storage
             switch add(lt(slength, 32), lt(newlength, 32))
-                case 2 {
-                    // Since the new array still fits in the slot, we just need to
-                    // update the contents of the slot.
-                    // uint256(bytes_storage) = uint256(bytes_storage) + uint256(bytes_memory) + new_length
-                    sstore(
-                        _preBytes_slot,
-                        // all the modifications to the slot are inside this
-                        // next block
+            case 2 {
+                // Since the new array still fits in the slot, we just need to
+                // update the contents of the slot.
+                // uint256(bytes_storage) = uint256(bytes_storage) + uint256(bytes_memory) + new_length
+                sstore(
+                    _preBytes_slot,
+                    // all the modifications to the slot are inside this
+                    // next block
+                    add(
+                        // we can just add to the slot contents because the
+                        // bytes we want to change are the LSBs
+                        fslot,
                         add(
-                            // we can just add to the slot contents because the
-                            // bytes we want to change are the LSBs
-                            fslot,
-                            add(
-                                mul(
-                                    div(
-                                        // load the bytes from memory
-                                        mload(add(_postBytes, 0x20)),
-                                        // zero all bytes to the right
-                                        exp(0x100, sub(32, mlength))
-                                    ),
-                                    // and now shift left the number of bytes to
-                                    // leave space for the length in the slot
-                                    exp(0x100, sub(32, newlength))
+                            mul(
+                                div(
+                                    // load the bytes from memory
+                                    mload(add(_postBytes, 0x20)),
```

Comment:
it's important for the tracebility of the changes that the code that is not changed, it does not appear in the diff. Try to shut down the linter, and reorganize the parts that you do not change